deps/leveldb/leveldb.gyp

```diff
@@ -26,9 +26,13 @@
     ]
   , 'conditions': [
         ['OS == "win"', {
+            'conditions': [
+                ['MSVS_VERSION != "2015" and MSVS_VERSION != "2013"', {
```

Comment:
> Also does this now mandate that the --msvs_version has to be passed as part of the build? 

No. `MSVS_VERSION` is defined by gyp based on whatever version of VS is selected by gyp etc..

> @obastemur I also noted here #213 (comment) that there is a patch to go into /deps/leveldb/port-libuv/port_uv.h
Did you come across that?

Actually I've changed a lot for UWP / VS2015 (see Level/leveldown-mobile) but the original issue is just because of `stdint.h` so the rest (if there is any) might be subject to another PR.

> It also looks like this commit 29282aa from @appelgriebsch is trying to fix the same issue.

IMHO that's not a solution. (shadowing stdint.h)

I've confirmed VS2013. (Don't know VS2012 so didn't add here)